### PR TITLE
Scene config multisource multilabel

### PR DESCRIPTION
**Pull request by pmeier82-zz**
- Test Script and FeatureCreator for number of sources detection
- fix for empty parameter AFE requests

### CODE REVIEW

File: src/+FeatureCreators/FeatureSetNSrcDetection.m [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            
+            % afeIdx 2: ITD
+            itd = obj.makeBlockFromAfe( 2, 1, ...
+                @(a)(compressAndScale( a.Data, 0.33 )), ...
+                {@(a)(a.Name),@(a)([num2str(numel(a.cfHz)) '-ch']),@(a)(a.Channel)}, ...
+                {@(a)(strcat('t', arrayfun(@(t)(num2str(t)),1:size(a.Data,1),'UniformOutput',false)))}, ...
+                {@(a)(strcat('f', arrayfun(@(f)(num2str(f)),a.cfHz,'UniformOutput',false)))} );
+            x = obj.concatFeats( x, obj.reshape2featVec( itd ) );
+            
+            % afeIdx 3: onset strengths
```
Comment (kashefy) [inline src/+FeatureCreators/FeatureSetNSrcDetection.m:80]: Doesn't look like this is a 3rd request. Just a repeat of the itd request.

Comment (pmeier82-zz) [reply src/+FeatureCreators/FeatureSetNSrcDetection.m:80]: True, no third request (yet). The Feature Creator is not finished. You can cherry pick the stuff you want to merge..

File: src/+DataProcs/AuditoryFEmodule.m [hunk 2]
```diff
@@ -140,11 +140,15 @@ function process( obj, wavFilepath )
         %% ----------------------------------------------------------------
         
         function s = parameter2struct( p )
+            s = struct();
```
Comment (ivo--t) [inline src/+DataProcs/AuditoryFEmodule.m:143]: Ok, of course :). This line is what was missing in the case of zero map count.

File: src/+DataProcs/AuditoryFEmodule.m [hunk 3]
```diff
@@ -140,11 +140,15 @@ function process( obj, wavFilepath )
...
         function s = parameter2struct( p )
+            s = struct();
             k = p.map.keys;
             v = p.map.values;
             for ii = 1 : p.map.Count
                 s.(k{ii}) = v{ii};
             end
+            if length(fields(s)) < 1
+                s.warning = 'no parameters given!';
+            end
```
Comment (ivo--t) [inline src/+DataProcs/AuditoryFEmodule.m:151]: However, these three are unnecessary. Please remove them and then I'll merge.

Comment (pmeier82-zz) [reply src/+DataProcs/AuditoryFEmodule.m:151]: Ok, will change.

**Pull request closed without merging**